ui/app/TokenList.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Chaosnet supported tokens",
+  "tokens": [
+    {
+      "chainId": 10,
+      "address": "0x7F5c764cBc14f9669B88837ca1490cCa17c31607",
+      "name": "USDCoin",
+      "decimals": 6,
+      "symbol": "USDC.e"
+    },
+    {
+      "chainId": 42161,
+      "address": "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
+      "name": "USDCoin",
+      "decimals": 6,
+      "symbol": "USDC"
+    },
+    {
+      "chainId": 58008,
```

sorry, this was the wrong chain ID
```suggestion
      "chainId": 11155111,
```